### ISSUE TITLE
zero values not rendering

### ISSUE THREAD

**Issue by benmonro** [yahoo/intl-messageformat#144]
i have a use case like this:

``` javascript
var intl = require("intl-messageformat");

var myThing = new intl("{counter} times", "en-us");
console.log(myThing.format({counter:0}))
```

the result of this will be " times" but I'd expect it to be "0 times"

is this a bug or on purpose?

**Comment by mimecuvalo** [comment]
The bug is in compiler.js
```
StringFormat.prototype.format = function (value) {
    if (!value) {
        return '';
    }
```

**Comment by jasonmit** [comment]
Please read this discussion https://github.com/yahoo/intl-messageformat/pull/136

**Comment by mimecuvalo** [comment]
I see - thanks!

**Comment by jasonmit** [comment]
Should be fixed in 2.0.0
